lib/c-api/build.rs

```diff
@@ -389,3 +390,19 @@ fn exclude_items_from_wasm_c_api(builder: Builder) -> Builder {
         .exclude_item("wasmer_engine_t")
         .exclude_item("wat2wasm")
 }
+
+fn build_inline_c_env_vars() {
+    let mut shared_object_dir = PathBuf::from(env::var("CARGO_MANIFEST_DIR").unwrap());
+    shared_object_dir.pop(); // removes `c-api`
+    shared_object_dir.pop(); // removes `lib`
```

Comment:
Optional:
```suggestion
    assert_eq!(shared_object_dir.pop(), "c-api".into());
    assert_eq!(shared_object_dir.pop(), "lib".into());
```
(note: code not actually tested) The idea is to check this so that it fails noisily instead of silently causing surprises down the line.

Reply:
That's an excellent idea!

Reply:
`pop` returns `bool`. The solution is to do this:

```
assert_eq!(shared_object_dir.file_name(), Some(OsStr::new("c-api")));
shared_object_dir.pop();
```

and so on.